packages/cozy-webview/src/api/services/WebviewService.ts

```diff
@@ -0,0 +1,24 @@
+import { ChildHandshake, Connection, RemoteHandle } from 'post-me'
+
+import { NativeMethods } from '@api/models/methods'
+import { WebviewMessenger } from '@api/services/WebviewMessenger'
+import { WebviewWindow } from '@api/models/environments'
+
+export class WebviewService {
+  private connection?: Connection
+  private messenger: WebviewMessenger
+  private remoteHandle?: RemoteHandle<NativeMethods>
+
+  constructor(windowRef: WebviewWindow, messengerService = WebviewMessenger) {
+    this.messenger = new messengerService(windowRef)
+    void this.init()
+  }
+
+  public init = async (): Promise<void> => {
```

Comment:
Shouldn't init be private or at least protected? Are we supposed to call it from outside?